src/main/java/io/supertokens/webserver/api/core/NotFoundOrHelloAPI.java

```diff
@@ -43,6 +43,11 @@ public String getPath() {
         return "/";
     }
 
```

Comment:
what changed from an older version that this is required, also if you visit `/recipe/abc`, it should hit this route and return a 404.